src/engine/ruleset/integrations/system/README.md

```diff
@@ -0,0 +1,58 @@
+# System syslog and auth events
+
+
+|   |   |
+|---|---|
+| event.module | system |
+
+
+The system module collects and parses logs created by the system logging service of common Unix/Linux based distributions.
+
+System-syslog logs include standar services syslog logging logs
+System-auth logs include auth logs like sudo, ssh, and cron logs
+
+
+## Compatibility
+
+This integration was tested with logs from OSes like Ubuntu 12.04, Centos 7, and macOS Sierra.
+
+## Configuration
+
+This integration ingests logs from /var/log/syslog and /var/log/auth.log (the location of the files
+can be configured in the localfile configuration).
+
+As mentioned above, we use the logcollector source localfile to ingest the logs from both files.
+
+Adding to the ossec.conf file in the monitored agent the following blocks:
+```xml
+  <localfile>
+    <location>/var/log/messages</location>
+    <log_format>syslog</log_format>
+    <label key="event.module">system</label>
+  </localfile>
+  <localfile>
+    <location>/var/log/secure</location>
+    <log_format>syslog</log_format>
+    <label key="event.module">system</label>
+  </localfile>
+  <localfile>
+  <location>/var/log/auth.log</location>
+    <log_format>syslog</log_format>
+    <label key="event.module">system</label>
+  </localfile>
+  <localfile>
+    <location>/var/log/kern.log</location>
+    <log_format>syslog</log_format>
+    <label key="event.module">system</label>
+  </localfile>
+
+```
```

Comment:
This type of configuration does not exist. It is not possible to use `syslog` format with the `label` option.
```suggestion
```xml
  <localfile>
    <location>/var/log/messages</location>
    <log_format>syslog</log_format>
  </localfile>
  <localfile>
    <location>/var/log/secure</location>
    <log_format>syslog</log_format>
  </localfile>
  <localfile>
  <location>/var/log/auth.log</location>
    <log_format>syslog</log_format>
  </localfile>
  <localfile>
    <location>/var/log/kern.log</location>
    <log_format>syslog</log_format>
  </localfile>
```
```

Reply:
Done !